src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -4,6 +4,8 @@
     const SELECTOR_BOOKMARK_LOCATION_INPUT = '#location_update_bookmark_location';
     const SELECTOR_BOOKMARK_WRAPPER = '.ez-add-to-bookmarks';
 
+    const CLASS_BOOKMARK_CHECKED = 'ez-add-to-bookmarks--checked';
```

Comment:
Please, no extra lines before this `const`

Reply:
You're using this CSS class once. There's no need to store it in a variable.

Reply:
It was when you were using it twice. There is no need to cache string when you use it once but it's also not a mistake, so it can stay. :)